integration_tests/dbt_project.yml

```diff
@@ -29,7 +29,7 @@ vars:
     jira_user_identifier: "user"
     jira_version_identifier: "version"
 
-    issue_field_history_columns: ['summary', 'components', 'story points']
+  issue_field_history_columns: ['summary', 'components', 'story points']
```

Comment:
@fivetran-avinash is this made into a global variable? In that case I think we should prefix it with "jira__"